crates/types/src/rpc/estimate_message_fee.rs

```diff
@@ -1,40 +1,46 @@
 use std::sync::Arc;
 
 use blockifier::transaction::transactions::L1HandlerTransaction;
-use serde::{Deserialize, Serialize};
 use starknet_api::core::EntryPointSelector;
 use starknet_api::transaction::Calldata;
 use starknet_rs_core::types::requests::EstimateMessageFeeRequest;
 use starknet_rs_core::types::{
-    BlockId as SrBlockId, FeeEstimate, MsgFromL1 as SrMsgFromL1, MsgFromL1,
+    BlockId as SrBlockId, FeeEstimate, MsgFromL1 as SrMsgFromL1, MsgFromL1, PriceUnit,
 };
+use starknet_rs_ff::FieldElement;
 
 use crate::error::DevnetResult;
 use crate::felt::Felt;
 use crate::rpc::block::BlockId;
 use crate::rpc::eth_address::EthAddressWrapper;
 use crate::{impl_wrapper_deserialize, impl_wrapper_serialize};
 
-#[derive(Debug, Clone, Deserialize, Serialize)]
-#[serde(tag = "unit")]
-pub enum FeeEstimateWrapper {
-    #[serde(rename = "WEI")]
-    WEI(FeeEstimate),
-    #[serde(rename = "FRI")]
-    STRK(FeeEstimate),
+#[derive(Clone, Debug)]
+pub struct FeeEstimateWrapper {
+    inner: FeeEstimate,
 }
```

Comment:
Why do we need the wrapper?

Reply:
before implementing 0.6 rpc spec it was used with the wrapper. But due to the fact that starknet-rs didnt implemented 0.6 spec yet, I had to change it. Now starknet-rs implemented it, so I switched it back to how it looked like before 0.6